src/composables/use-media-query.js

```diff
@@ -0,0 +1,35 @@
+import { useEventListener } from '~/composables/use-event-listener'
+import { computed, ref } from '@nuxtjs/composition-api'
+
+/**
+ * Use a media query.
+ * @param {Ref<string> | string} query
```

Comment:
I used the same code here 😆 : https://github.com/WordPress/openverse-frontend/pull/445/files#diff-8408da4942f82bca81affaff29a72e22ee03a7d3c7dc3966684abec20418d8fdR1-R38
Window needs to be handled correctly in SSR.

Reply:
Awesome, I'll pull in your implementation :)